app/views/spree/shared/_reviews.html.erb

```diff
@@ -1,12 +1,8 @@
 <div id="reviews">
-  <h3 class="product-section-title"><%= Spree.t(:reviews) %></h3>
   <% if Spree::Reviews::Config[:include_unapproved_reviews] == false and @product.reviews.approved.count == 0 %>
-    <p class="reviews-none"><%= Spree.t(:no_reviews_available) %></p>
+  	<%= render 'spree/reviews/stars', stars: 0 %>
   <% else %>
     <%= render 'spree/shared/rating', product: @product, review: 0 %>
-    <% for review in (Spree::Reviews::Config[:track_locale] ? @product.reviews.localized(I18n.locale) : @product.reviews).default_approval_filter.preview %>
-      <%= render 'spree/shared/review', review: review %>
-    <% end %>
   <% end %>
-  <%= link_to Spree.t(:write_your_own_review), new_product_review_path(@product), class: 'btn btn-default', rel: 'nofollow' %>
+  <a class="write_review_button" href="" onclick="$('a[href=\'#tab-review\']').trigger('click'); return false;"><%= Spree.t('pages.common.review_new')%></a>
```

Comment:
Remove the inline js and make a separate review.js file inside assets and put all the inline js there.